src/graphql/mongoose_graphql_account_helper.erl

```diff
@@ -0,0 +1,26 @@
+-module(mongoose_graphql_account_helper).
+
+-export([format_result/2, make_error/2, make_error/3]).
+
+-include("mongoose_graphql_types.hrl").
+
+-spec format_result(InResult, Context) -> OutResult when
+      InResult :: {atom(), string() | binary() | integer()},
```

Comment:
I think we could get `iolist` here which is not covered by the spec (because `io_lib:format` is used), could you check this?